src/layers/RouteLayer/RouteLayer.js

```diff
@@ -192,11 +195,12 @@ class RouteLayer extends CasaLayer {
       const lineStart = feature.getGeometry().getFirstCoordinate();
       const lineEnd = feature.getGeometry().getLastCoordinate();
 
-      styleArray.push(RouteLayer.getCircleStyle(lineStart));
-      styleArray.push(RouteLayer.getCircleStyle(lineEnd));
+      /* Set circle icon if it is the first sequence of a route */
+      if (isStart) styleArray.push(RouteLayer.getCircleStyle(lineStart));
```

Comment:
Use brackets. Putting `isStart` in a variable and renaming it has no effect besides reducing readability.
Directly do `if (feature.get('isStart')`

Reply:
done